parsl-introduction.ipynb

```diff
@@ -229,7 +233,7 @@
     "\n",
     "app_future = hello()\n",
     "\n",
-    "# Check if the app_future is resolved\n",
+    "# Check if the app_future is resolved, which it won't be\n",
```

Comment:
which it won't be if we are fast enough? won't be yet? won't be because we run this before the 5s are up?